snapcraft/bin/cvescan.py

```diff
@@ -2,6 +2,7 @@
 
 import sys
 import os
+import os.path
```

Comment:
do both os and os.path need to be imported?

Reply:
apparently not. I'll remove it.